plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/plugin.xml

```diff
@@ -61,9 +61,6 @@
         id="com.google.cloud.tools.eclipse"
         name="Google Cloud Platform"
         class="com.google.cloud.tools.eclipse.properties.GooglePropertyPage">
-       <enabledWhen>
-         <reference definitionId="com.google.cloud.tools.eclipse.appengine.onlyInGCPProjects" />
```

Comment:
One of the things I found annoying about GPE was how it contributed a `Google` properties page to **everything**.  We should only show property pages where they belong.

The problem with `onlyInGCPProjects` is it's not currently easily extensible for new project types without changing its definition.  We can come up with ways to fix that, but we should continue to use it.

How to make `onlyInGCPProjects` extensible? If our facets are mutually exclusive, we could install a nature. Or we could create a custom property test that checks the project's facets IDs matches `com.google.cloud.tools.**`, or against an extension point listing our known GCP facets.